packages/replayio/src/commands/record.ts

```diff
@@ -20,7 +24,26 @@ registerCommand("record", { checkForRuntimeUpdate: true, requireAuthentication:
 async function record(url: string = "about:blank") {
   const prevRecordings = await getRecordings();
 
-  await launchBrowser(url, { processGroupId: uuid() });
+  try {
+    await launchBrowser(url, { processGroupId: uuid() });
+  } catch (error) {
+    if (error instanceof ProcessError) {
+      console.log("\nSomething went wrong while recording. Try again.");
+
+      // TODO [PRO-235] Upload recorder crash data somewhere
```

Comment:
Filed PRO-235 for follow up here because I think it's worth doing eventually/soon.